src/main/java/com/launchacademy/reviews/controllers/HomeController.java

```diff
@@ -10,14 +10,16 @@
     "/pizza-styles/{id}", 
     "/pizza-styles/new", 
     "/reviews/new",
+    "/reviews/{id}/edit",
     "/404"
   })
+  
   public String forward() {
     return "forward:/";
   }
 
-    @RequestMapping(value = "/**/{path:[^\\\\.]*}")
-    public String redirect() {
-        return "forward:/404";
-    }
```

Comment:
This is a little weird that the code is the same but it's showing as a change. Was there an intentional spacing/tab change or was it not touched?

Reply:
It was a spacing change after pulling from main